### ISSUE TITLE
keepalive onPingResponse subscribe response doesnt work right 

### ISSUE THREAD

**Issue by hakim-dc** [HackedByChinese/ng2-idle#118]
**I'm submitting a ...**
```
[X] bug report

**Current behavior**
this subscription must return the web-service response used to update session expiry time
"keepalive.onPingResponse.subscribe(response => console.log(response);"

**Expected behavior**
after subscribe onPingResponce the response.status must return the  status of invoked 
http request in "response.status"
example: 200 for success 

**Minimal reproduction of the problem with instructions**
the main problem is that this subscription always return first a response that contain a type equal to 0
after that it return the real webservice response (response.status not null ) for each keepalive webservice
 pingRequest  but in the other hand when the webservise return request.status different to 200 
(not success) we get first also a response that contain a type equal to 0 and no other response.


**What is the motivation / use case for changing the behavior?**
my expected behavior is that method return only response that contain the http request status in case of 
success or fail of web-service invocation and remove the responce that contain type equal to 0
 "response.type = 0" 

**Please tell us about your environment:**
Ubuntu, visualCode, npm
* **@ng-idle version:** 6.0.0-beta.3


* **Angular version:** 6.x


* **Browser:** all


* **Language:** TypeScript 2.7.2
* **Node:** `node --version` = v10.9.0

**Comment by JIGR1969** [comment]
I'm having a similar issue, looking at the environment, mine is identical.

```
        // Sets the ping interval to 30 minutes, backend times out after 60 minutes.
        keepalive.interval(20);
        keepalive.request(this.pingUrl);
        keepalive.onPing.subscribe((response) => {
            
            // Set a none existent element to the last ping date.
            this.lastPing = new Date(); 
            console.log("KeepAlive PING sent @ " + this.lastPing);
            
            console.log(response);
        });
```

That is the code that I'm using, looking at the console within Chrome I can see that two PINGS are being sent. One is an OPTIONS call and the other is a GET, the latter being the correct one I'm guessing. not sure why the first one is being called. The first one is initiated by 'zone.js' and the second one by 'other'.

An error then occurs with the message :
'message: "Http failure response for http://172.23.100.58/api/auth/ping: 300 Multiple Choices"'

**Comment by mmubasher** [comment]
@HackedByChinese Kindly review and merge [pull request](#120 )

**Comment by mmubasher** [comment]
Can you please release this on npm as we need this change available? I tried `npm run bundle` and it created the packages but deployments will only work with the npm packages `@ng-idle/core` and `@ng-idle/keepalive`. It would be great if you can release a package soon. Thank you

**Comment by HackedByChinese** [comment]
it's published as 6.0.0-beta.4
